### ISSUE TITLE
Rule refactoring 1 - CC (carding)

### ISSUE THREAD

**Issue by masterl** [fsocietybrasil/rules#7]
Texto atual:

> CC (carding) totalmente proibido.
> Penalidade - Perma BAN e denúncia pra PF.

**Comment by masterl** [comment]
Talvez fosse bom criar uma regra mais geral para fraudes.
Aí seria uma questão de simplesmente listar mais casos quando necessário.

**Comment by yuriifreire** [comment]
Apenas incrementar mais casos quando necessário, de resto, mantém.

**Comment by masterl** [comment]
Concordo, mas me referia a renomear de ***CC (carding)*** para ***fraudes*** ou algo do tipo.
Para ficar mais claro que essa regra abarca esse tipo de coisa.

**Comment by yuriifreire** [comment]
Por mim, ta ok. Fica ate melhor para quando pegarmos membros que nao sabem o que eh CC
